Paco/src/com/pacoapp/paco/net/ServerAddressBuilder.java

```diff
@@ -17,7 +17,7 @@ public static String createServerUrl(String serverAddress, String path) {
   }
 
   public static boolean isLocalDevelopmentServerAddress(String serverAddress) {
-    return serverAddress.contains("10.0.2.2");
+    return serverAddress.contains("10.0.2.2") || serverAddress.contains("127.0.0.1");
```

Comment:
Hi Bram,

Thanks for reminding me. I had completely forgotten about that :-)

Best,
Bob

Http://www.pacoapp.com/

> On Aug 5, 2016, at 2:54 PM, Bram Bonné <notifications@github.com> wrote:
> 
> In Paco/src/com/pacoapp/paco/net/ServerAddressBuilder.java:
> 
> > @@ -17,7 +17,7 @@ public static String createServerUrl(String serverAddress, String path) {
> >    }
> >  
> >    public static boolean isLocalDevelopmentServerAddress(String serverAddress) {
> > -    return serverAddress.contains("10.0.2.2");
> > +    return serverAddress.contains("10.0.2.2") || serverAddress.contains("127.0.0.1");
> Hi Bob,
> 
> I understand that this isn't immediately clear. A very easy way to get your phone to connect to a testserver running on the machine it's connected to, is by just forwarding its local port to a port on the machine, like this:
> 
> adb reverse tcp:8080 tcp:8080
> Then the phone, whatever machine its connected to through adb, can just use 127.0.0.1:8080 as the server address to connect to the host. I'll add a comment in the code to explain this too :-)
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>